apps/derive/elpi/eqK.elpi

```diff
@@ -100,16 +100,16 @@ main-constructor Lno Prefix K Kty C :- do! [
   coq.env.add-const Name R RT @opaque! EqK,
 
   % add the clause to the db
-  C = (eqK-db K (global (const EqK)) :- !),
+  C = (pi u\ eqK-db K (global (const EqK) u) :- !),
```

Comment:
This is maybe the most relevant change: storing univ-instance in a DB is not really what you want so one has to quantify it so that the clause works for any univ-instance.